api/app/Providers/RouteServiceProvider.php

```diff
@@ -57,7 +57,14 @@ public function boot()
     protected function configureRateLimiting()
     {
         RateLimiter::for('api', function (Request $request) {
-            return Limit::perMinute(60)->by(optional($request->user())->id ?: $request->ip());
+            return Limit::perMinute(100)->by(optional($request->user())->id ?: $request->ip())->response(function () {
```

Comment:
I agree that 100 is probably too low.  600 maybe?  Since this is in multiple places and we probably will want to tune it could it be factored out to a config and added to the .env file?

Reply:
Sure, 600 for both api and graphql, or 600 for graphql and api remains at 100? Should the Acceptance Criteria be updated for the attached issue as well?

I like the idea of environment variables. What do you think of:

- RATE_LIMIT_GRAPHQL
- RATE_LIMIT_API